rest/admin_api_test.go

```diff
@@ -4119,3 +4120,108 @@ func TestEmptyStringJavascriptFunctions(t *testing.T) {
 	assert.NoError(t, resp.Body.Close())
 	assert.Equal(t, http.StatusCreated, resp.StatusCode)
 }
+
+func TestGroupIDReplications(t *testing.T) {
+	t.Skip("Disabled until finished") // TODO: Finish test
+	if base.UnitTestUrlIsWalrus() || !base.TestUseXattrs() {
+		t.Skip("This test only works against Couchbase Server with xattrs enabled")
+	}
+	defer base.SetUpTestLogging(base.LevelInfo, base.KeyAll)()
+
+	// Create test buckets to replicate between
+	passiveBucket := base.GetTestBucket(t)
+	defer passiveBucket.Close()
+
+	activeBucket := base.GetTestBucket(t)
+	defer activeBucket.Close()
+
+	// Set up passive bucket RT
+	rt := NewRestTester(t, nil)
+	defer rt.Close()
+
+	// Make rt listen on an actual HTTP port, so it can receive replications
+	srv := httptest.NewServer(rt.TestAdminHandler())
+	defer srv.Close()
+	passiveDBURL, err := url.Parse(srv.URL + "/db")
+	require.NoError(t, err)
+
+	// Start SG nodes for default group, group A and group B
+	groupIDs := []string{"", "GroupA", "GroupB"}
+	var adminHosts []string
+	var serverContexts []*ServerContext
+	for i, group := range groupIDs {
+		serverErr := make(chan error, 0)
+
+		config := bootstrapStartupConfigForTest(t)
+		portOffset := i * 10
+		adminInterface := fmt.Sprintf("127.0.0.1:%d", 4985+bootstrapTestPortOffset+portOffset)
+		adminHosts = append(adminHosts, "http://"+adminInterface)
+		config.API.PublicInterface = fmt.Sprintf("127.0.0.1:%d", 4984+bootstrapTestPortOffset+portOffset)
+		config.API.AdminInterface = adminInterface
+		config.API.MetricsInterface = fmt.Sprintf("127.0.0.1:%d", 4986+bootstrapTestPortOffset+portOffset)
+		config.Bootstrap.ConfigGroupID = group
+
+		sc, err := setupServerContext(&config, true)
+		require.NoError(t, err)
+		serverContexts = append(serverContexts, sc)
+		defer func() {
+			sc.Close()
+			require.NoError(t, <-serverErr)
+		}()
+		go func() {
+			serverErr <- startServer(&config, sc)
+		}()
+		require.NoError(t, sc.waitForRESTAPIs())
+
+		// Set up db config
+		resp := bootstrapAdminRequestCustomHost(t, http.MethodPut, adminHosts[i], "/db/",
+			fmt.Sprintf(
+				`{"bucket": "%s", "num_index_replicas": 0, "use_views": %t, "import_docs": true, "sync":"%s"}`,
+				passiveBucket.GetName(), base.TestsDisableGSI(), channels.DefaultSyncFunction,
+			),
+		)
+		assert.Equal(t, http.StatusCreated, resp.StatusCode)
+	}
+
+	// Start replicators
+	for i, group := range groupIDs {
+		channel := "chan" + group
+		replicationID := "repl" + group
```

Comment:
Similar to the previous test - since we don't have any collision on replication IDs, it seems like this test might have passed prior to the namespacing enhancements in this PR?